### PR TITLE
Add media capture depth dictionary test

### PR DESCRIPTION
**Pull request by xiuqijix**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-09-13 01:15:53
Finished: 2017-09-13 01:20:35


<h2>Failing Jobs</h2>
<ul>

<li>safari:10.0</li>

<li>MicrosoftEdge:14.14393</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/18479)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/274853397)

**Comment by foolip** [comment]
Hi @kaixinjxq! I guess you reasonably expected that if you submit a PR someone will eventually come and take a look. Unfortunately that's not quite the case, especially when the status:needs-owners gets added you'll have to go hunting for someone to review.

Who would be a good reviewer for this and https://github.com/w3c/web-platform-tests/pull/7457?

**Comment by kaixinjxq** [comment]
@huningxin, @astojilj are you happy to be the owner to help review depth tests?

**Comment by kaixinjxq** [comment]
This PR is blocked on the required "Travis CI - Pull Request" check after #14499. In order to trigger it, I will close and reopen this PR.

**Comment by astojilj** [comment]
@kainino0x thanks. I don't know how I missed this. 
LGTM

File: mediacapture-depth/OWNERS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@huningxin
```
Comment (foolip) [inline mediacapture-depth/OWNERS]: Can you make this a META.yml file instead?

**Review by Honry — APPROVED** [review]
LGTM, but I think most tests could be covered by idlharness.js if we have this feature https://github.com/web-platform-tests/wpt/issues/9106 implemented. :)

**Pull request merged**